src/plugins/MessagesPlugin/MessageKeys.ts

```diff
@@ -0,0 +1,4 @@
+type MessageKeys =
+	'wikibaselexeme-newlexeme-lemma';
```

Comment:
Wait, isn't `MessageKeys` used to annotate types for the plugin's `get()` and `getUnescaped`? I'm not sure a literal alias is the way to go here, unless I'm missing something

Reply:
Mh, I maybe should have added more. My goal is to extend this as a list of all allowed messages:

```ts
type MessageKeys =
	'wikibaselexeme-newlexeme-lemma'
	| 'foo'
	| 'bar';

```

Reply:
That is, the type is intended to be "one of these exact literal strings"

Reply:
> Ah, yeah I got that, but I thought this would be derived from an object similar to what we added in PR #33 (exported from a file ofc, and not just ad-hoc in index.html):
> 
> https://github.com/wmde/new-lexeme-special-page/blob/5492aa7ddae6ec22b56d8d499f650d604fb5d5e2/index.html#L13-L15

But, these messages are being stored outside of this git repository, in https://github.com/wikimedia/mediawiki-extensions-WikibaseLexeme/blob/master/i18n/en.json

I don't think duplicating this copy is a good idea. They are bound to diverge.

Reply:
Yes, I also understand that. I just think having a version of the copy available for local development could be useful, especially for visual tasks like copy on a button. It doesn't have to stay in sync.

Reply:
Yeah, having mocks for the copy makes sense, I agree with that. But mocks are no source for types for production code. Rather the other way around, the mock could be typed with the production types:

```ts
 const mockMessages: Record<MessageKeys, string> = { 
 	'wikibaselexeme-new-lexeme-submit': 'Create Lexeme', 
 };
 ```
 
 or something like that.

Reply:
> But mocks are no source for types for production

Agreed, just wanted to prevent having to synchronize the same keys everywhere.